multihost_test/bz_automation/test_automation.py

```diff
@@ -310,3 +310,29 @@ def test_pam_access_account(self, multihost, create_localusers, bkp_pam_config):
         client.run_command("sh /tmp/authentication.sh")
         log_str = multihost.client[0].get_file_contents("/var/log/secure").decode('utf-8')
         assert "LOCAL" not in log_str
+
+    def test_cve_pam_access(self, multihost, bkp_pam_config, create_localusers):
+        """
+        :title: pam: Improper Hostname Interpretation in pam_access Leads to Access Control Bypass
```

Comment:
We already know that this is a PAM issue, so no need to mention it in the title. In addition, remove the capital letters